three-cards-divination/src/test/java/com/cyber/mysticism/tarot/controller/ThreeCardsDivinationControllerFunctionalTest.java

```diff
@@ -22,6 +22,7 @@
 @AutoConfigureMockMvc
 class ThreeCardsDivinationControllerFunctionalTest {
 
+    public static final String AUTH_HEADER = "Basic dXNlcjplbWFpbA==";
```

Comment:
**Suggestion:** Avoid hardcoding the `AUTH_HEADER` value in your tests. Consider using a method to generate this value. [best practice]
```suggestion
    public static final String AUTH_HEADER = generateAuthHeader("user", "email");
```